tests/e2e/basicsPageFunctionality.js

```diff
@@ -4,11 +4,16 @@ module.exports = {
   '@tags': ['parallel'],
   'User Logs in with Existing Credentials'(browser) {
     const loginPage = browser.page.loginPage();
-    const loginForm = loginPage.section.loginForm;
+    //const loginForm = loginPage.section.loginForm;
```

Comment:
just delete these lines if they're not being used rather than commenting them out.

Reply:
Deleted!